packages/manager/modules/pci/src/projects/project/kubernetes/details/service/service.routing.js

```diff
@@ -46,6 +49,9 @@ export default /* @ngInject */ ($stateProvider) => {
           { kubeId, projectId },
         ),
 
+      isVersionSupported: /* @ngInject */ (clusterMinorVersion, versions) =>
+        indexOf(versions, toString(clusterMinorVersion)) > -1,
```

Comment:
What do you think about
```suggestion
        versions.includes(toString(clusterMinorVersion)),
```
?

Reply:
I was thinking Lodash is our first preference.

Reply:
We decided together (you included) that you could use whatever you wanted during the 2019-10-17 tech talk (you can read the summary with the document sharing service 😄 )

Lodash also has `includes` if you want to stick to Lodash: https://lodash.com/docs/4.17.15#includes